core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -221,10 +224,33 @@ export class ContributorDashboardPageComponent
 
     this.activeTabName = 'myContributionTab';
 
-    // TODO(#13015): Remove use of unknown as a type.
+    // TODO(sll): Use constant values from
```

Comment:
Why does this refer to me? All TODOs should refer to an issue number that provides full details.

Also is the following the only occurrence of the "fully-detailed" version of this constant? Just wanted to make sure that we're not repeating the same large dict in multiple places.

Reply:
Yup working with Srijan on this type issue. Unable to fix this by my know without repeating of same dict. This issue also i'm dealing in srijan comments, so you can resolve it here. Will take care of this.

Reply:
Er, I can't really resolve it as long as there's a "TODO(sll)" here... As codeowner I need to check the final state of all the files I code-own before I can sign off.

Couldn't you do the thing you mention in the TODOs (i.e. refer to where it's defined rather than redefining it)?

Anyway, feel free to ping again on this thread once the "duplicate constants" issue has been resolved, and I'll take another look. Thanks!

Reply:
Done, point to issue #13015.

Reply:
The comment still says "TODO(sll)".